pinctrl/hw_pwm.go

```diff
@@ -76,7 +76,8 @@ const (
 
 type pwmDevice struct {
 	chipPath string
-	line     int
+	pwmLine  int
+	gpioLine int
```

Comment:
originally the field was just called line so I thought I should keep the same naming, but it would prob be better if I used `pwmID` and `line` instead